example/src/App.tsx

```diff
@@ -95,6 +97,14 @@ const App = () => {
               title: "🎨 Skia",
             }}
           />
+          <Stack.Screen
```

Comment:
We don't need an extra example here, we can just add a gradient here: https://github.com/Shopify/react-native-skia/blob/main/example/src/Examples/API/Gradients.tsx

Reply:
It's not very visible on with the small tile but I will remove this from PR as soon as dither is added, just wanted to show the problem...

Reply:
ok. Now that we have the test we can remove it.